cmd/kubelet/app/options/globalflags.go

```diff
@@ -51,8 +51,8 @@ func normalize(s string) string {
 	return strings.Replace(s, "_", "-", -1)
 }
 
-// register adds a flag to local that targets the Value associated with the Flag named globalName in global
-func register(global *flag.FlagSet, local *pflag.FlagSet, globalName string) {
+// Register adds a flag to local that targets the Value associated with the Flag named globalName in global
+func Register(global *flag.FlagSet, local *pflag.FlagSet, globalName string) {
```

Comment:
Why exported? If not used outside of this package scope then no need to export(CapsCase)